package.json

```diff
@@ -2,45 +2,39 @@
   "name": "geniejs",
```

Comment:
I like the way this was sorted before better...

Reply:
I sorted it by using [`sort-package-json`](https://www.npmjs.com/package/sort-package-json).